Instead of keyboard name it showing wireless pid

**Information**
<!-- Do not bother opening an issue for version 0.9.2 or older, it is extremely outdated. Please update to the latest one and see if your issue persists. -->
- Solaar: 1.0.1 latest git from PPA repo
- Distribution: Linux Mint 19.3
- Kernel version (ex. `uname -srmo`): `Linux 5.5.9-050509-lowlatency x86_64 GNU/Linux`
- Output of `solaar show`:
```
Nano Receiver
  Device path  : /dev/hidraw3
  USB id       : 046d:c517
  Serial       : None
solaar: error: 'NoneType' object is not iterable

```

**Describe the bug**
I noticed in a new Solaar version that I have now listed 11 years old [Logitech S 510 Cordless Keyboard](https://support.logi.com/hc/en-us/articles/360023308154-S-510-Technical-Specifications).
Only it has battery status and instead of keyboard name it showing wireless pid.
Like in screenshots:
![Solaar](https://user-images.githubusercontent.com/1978777/76764745-a852ad00-6795-11ea-80d6-6b8a2bb2884c.png)

@pfps 
Now it won't work again.

<p>
</details>

<details>
<summary>solaar -dd show</summary>

```
20:18:21,099    DEBUG [MainThread] logitech_receiver.base: (3) <= w[10 FF 83B5 030000]
20:18:21,101    DEBUG [MainThread] logitech_receiver.base: (3) => r[10 FF 8F83 B50100]
20:18:21,101    DEBUG [MainThread] logitech_receiver.base: (3) device 0xFF error on request {83B5}: 1 = invalid SubID/command
20:18:21,101    DEBUG [MainThread] solaar.cli: [/dev/hidraw3] => <EX100Receiver27Mhz(/dev/hidraw3,3)>
20:18:21,104    DEBUG [MainThread] logitech_receiver.base: (4) <= w[10 FF 83B5 030000]
20:18:21,108    DEBUG [MainThread] logitech_receiver.base: (4) => r[10 FF 8F83 B50100]
20:18:21,108    DEBUG [MainThread] logitech_receiver.base: (4) device 0xFF error on request {83B5}: 1 = invalid SubID/command
20:18:21,108    DEBUG [MainThread] solaar.cli: [/dev/hidraw1] => <EX100Receiver27Mhz(/dev/hidraw1,4)>
20:18:21,245    DEBUG [MainThread] logitech_receiver.base: (3) <= w[10 FF 8102 000000]
20:18:21,253    DEBUG [MainThread] logitech_receiver.base: (3) => r[10 FF 8102 000200]
EX100 Receiver 27 Mhz
  Device path  : /dev/hidraw3
  USB id       : 046d:c517
  Serial       : None
20:18:21,253    DEBUG [MainThread] logitech_receiver.base: (3) <= w[10 FF 81F1 010000]
20:18:21,261    DEBUG [MainThread] logitech_receiver.base: (3) => r[10 FF 8F81 F10200]
20:18:21,261    DEBUG [MainThread] logitech_receiver.base: (3) device 0xFF error on request {81F1}: 2 = invalid address
  Has 2 paired device(s) out of a maximum of 4.
20:18:21,261    DEBUG [MainThread] logitech_receiver.base: (3) <= w[10 FF 8102 000000]
20:18:21,269    DEBUG [MainThread] logitech_receiver.base: (3) => r[10 FF 8102 000200]
20:18:21,269    DEBUG [MainThread] logitech_receiver.base: (3) <= w[10 FF 8100 000000]
20:18:21,277    DEBUG [MainThread] logitech_receiver.base: (3) => r[10 FF 8100 000100]
  Notifications: wireless (0x000100)
20:18:21,277    DEBUG [MainThread] logitech_receiver.base: (3) <= w[10 FF 83B3 000000]
20:18:21,285    DEBUG [MainThread] logitech_receiver.base: (3) => r[10 FF 8F83 B30100]
20:18:21,285    DEBUG [MainThread] logitech_receiver.base: (3) device 0xFF error on request {83B3}: 1 = invalid SubID/command
20:18:21,285    DEBUG [MainThread] logitech_receiver.base: (3) <= w[10 FF 8102 000000]
20:18:21,293    DEBUG [MainThread] logitech_receiver.base: (3) => r[10 FF 8102 000200]
20:18:21,293    DEBUG [MainThread] logitech_receiver.base: (3) <= w[10 FF 83B5 200000]
20:18:21,301    DEBUG [MainThread] logitech_receiver.base: (3) => r[10 FF 8F83 B50100]
20:18:21,301    DEBUG [MainThread] logitech_receiver.base: (3) device 0xFF error on request {83B5}: 1 = invalid SubID/command
20:18:21,305     INFO [MainThread] logitech_receiver.receiver: <EX100Receiver27Mhz(/dev/hidraw3,3)>: found new device 1 (3F00)

20:18:21,305    DEBUG [MainThread] logitech_receiver.base: (5) pinging device 1
20:18:21,305    DEBUG [MainThread] logitech_receiver.base: (5) <= w[10 01 001F 00005D]
20:18:21,309    DEBUG [MainThread] logitech_receiver.base: (5) => r[10 01 8F00 1F0100]
  1: Wireless Mouse EX100
     Device path  : /dev/hidraw7
     USB id       : 046d:3F00
     Codename     : EX100m
     Kind         : mouse
     Protocol     : HID++ 1.0
20:18:21,309    DEBUG [MainThread] logitech_receiver.base: (3) <= w[10 FF 83B5 200000]
20:18:21,317    DEBUG [MainThread] logitech_receiver.base: (3) => r[10 FF 8F83 B50100]
20:18:21,317    DEBUG [MainThread] logitech_receiver.base: (3) device 0xFF error on request {83B5}: 1 = invalid SubID/command
20:18:21,317    DEBUG [MainThread] logitech_receiver.base: (3) <= w[10 FF 83B5 300000]
20:18:21,325    DEBUG [MainThread] logitech_receiver.base: (3) => r[10 FF 8F83 B50100]
20:18:21,325    DEBUG [MainThread] logitech_receiver.base: (3) device 0xFF error on request {83B5}: 1 = invalid SubID/command
20:18:21,325    DEBUG [MainThread] logitech_receiver.base: (3) <= w[10 FF 83D5 000000]
20:18:21,333    DEBUG [MainThread] logitech_receiver.base: (3) => r[10 FF 8F83 D50100]
20:18:21,333    DEBUG [MainThread] logitech_receiver.base: (3) device 0xFF error on request {83D5}: 1 = invalid SubID/command
     Serial number: ?
20:18:21,333    DEBUG [MainThread] logitech_receiver.base: (5) <= w[10 01 81F1 010000]
20:18:21,341    DEBUG [MainThread] logitech_receiver.base: (5) => r[10 01 8F81 F10200]
20:18:21,341    DEBUG [MainThread] logitech_receiver.base: (5) device 0x01 error on request {81F1}: 2 = invalid address
20:18:21,341    DEBUG [MainThread] logitech_receiver.base: (3) <= w[10 FF 83B5 300000]
20:18:21,349    DEBUG [MainThread] logitech_receiver.base: (3) => r[10 FF 8F83 B50100]
20:18:21,349    DEBUG [MainThread] logitech_receiver.base: (3) device 0xFF error on request {83B5}: 1 = invalid SubID/command
     The power switch is located on the (unknown).
20:18:21,349    DEBUG [MainThread] logitech_receiver.base: (5) <= w[10 01 8100 000000]
20:18:21,357    DEBUG [MainThread] logitech_receiver.base: (5) => r[10 01 8100 3C0000]
     Notifications: roller V, mouse extra buttons, battery status, roller H (0x3C0000).
20:18:21,357    DEBUG [MainThread] logitech_receiver.base: (5) <= w[10 01 8101 000000]
20:18:21,365    DEBUG [MainThread] logitech_receiver.base: (5) => r[10 01 8101 420000]
     Features: special buttons, scroll accel (0x420000)
20:18:21,365    DEBUG [MainThread] logitech_receiver.base: (5) <= w[10 01 8107 000000]
20:18:21,373    DEBUG [MainThread] logitech_receiver.base: (5) => r[10 01 8107 050000]
     Battery: good, discharging.
20:18:21,373    DEBUG [MainThread] logitech_receiver.base: (3) <= w[10 FF 83B5 210000]
20:18:21,381    DEBUG [MainThread] logitech_receiver.base: (3) => r[10 FF 8F83 B50100]
20:18:21,381    DEBUG [MainThread] logitech_receiver.base: (3) device 0xFF error on request {83B5}: 1 = invalid SubID/command
20:18:21,381    ERROR [MainThread] logitech_receiver.device: failed to set fake EX100 wpid for device 2 of <EX100Receiver27Mhz(/dev/hidraw3,3)>
20:18:21,381    ERROR [MainThread] logitech_receiver.receiver: register_new_device
Traceback (most recent call last):
  File "/usr/share/solaar/lib/logitech_receiver/receiver.py", line 150, in register_new_device
    dev = Device(self, number, notification)
  File "/usr/share/solaar/lib/logitech_receiver/device.py", line 116, in __init__
    raise _base.NoSuchDevice(number=number, receiver=receiver, error='Unknown EX100 device')
logitech_receiver.base.NoSuchDevice: {'number': 2, 'receiver': <EX100Receiver27Mhz(/dev/hidraw3,3)>, 'error': 'Unknown EX100 device'}
20:18:21,382  WARNING [MainThread] logitech_receiver.receiver: <EX100Receiver27Mhz(/dev/hidraw3,3)>: looked for device 2, not found
20:18:21,382    DEBUG [MainThread] logitech_receiver.base: (3) <= w[10 FF 83B5 220000]
20:18:21,389    DEBUG [MainThread] logitech_receiver.base: (3) => r[10 FF 8F83 B50100]
20:18:21,389    DEBUG [MainThread] logitech_receiver.base: (3) device 0xFF error on request {83B5}: 1 = invalid SubID/command
20:18:21,392     INFO [MainThread] logitech_receiver.receiver: <EX100Receiver27Mhz(/dev/hidraw3,3)>: found new device 3 (6500)

20:18:21,393    DEBUG [MainThread] logitech_receiver.base: (6) pinging device 3
20:18:21,393    DEBUG [MainThread] logitech_receiver.base: (6) <= w[10 03 001A 0000BD]
20:18:21,397    DEBUG [MainThread] logitech_receiver.base: (6) => r[10 03 8F00 1A0100]
  3: Wireless Keyboard EX100
     Device path  : /dev/hidraw5
     USB id       : 046d:6500
     Codename     : EX100
     Kind         : keyboard
     Protocol     : HID++ 1.0
20:18:21,397    DEBUG [MainThread] logitech_receiver.base: (3) <= w[10 FF 83B5 220000]
20:18:21,405    DEBUG [MainThread] logitech_receiver.base: (3) => r[10 FF 8F83 B50100]
20:18:21,405    DEBUG [MainThread] logitech_receiver.base: (3) device 0xFF error on request {83B5}: 1 = invalid SubID/command
20:18:21,405    DEBUG [MainThread] logitech_receiver.base: (3) <= w[10 FF 83B5 320000]
20:18:21,413    DEBUG [MainThread] logitech_receiver.base: (3) => r[10 FF 8F83 B50100]
20:18:21,413    DEBUG [MainThread] logitech_receiver.base: (3) device 0xFF error on request {83B5}: 1 = invalid SubID/command
20:18:21,413    DEBUG [MainThread] logitech_receiver.base: (3) <= w[10 FF 83D5 000000]
20:18:21,421    DEBUG [MainThread] logitech_receiver.base: (3) => r[10 FF 8F83 D50100]
20:18:21,421    DEBUG [MainThread] logitech_receiver.base: (3) device 0xFF error on request {83D5}: 1 = invalid SubID/command
     Serial number: ?
20:18:21,421    DEBUG [MainThread] logitech_receiver.base: (6) <= w[10 03 81F1 010000]
20:18:21,429    DEBUG [MainThread] logitech_receiver.base: (6) => r[10 03 8F81 F10200]
20:18:21,429    DEBUG [MainThread] logitech_receiver.base: (6) device 0x03 error on request {81F1}: 2 = invalid address
20:18:21,429    DEBUG [MainThread] logitech_receiver.base: (3) <= w[10 FF 83B5 320000]
20:18:21,437    DEBUG [MainThread] logitech_receiver.base: (3) => r[10 FF 8F83 B50100]
20:18:21,437    DEBUG [MainThread] logitech_receiver.base: (3) device 0xFF error on request {83B5}: 1 = invalid SubID/command
     The power switch is located on the (unknown).
20:18:21,437    DEBUG [MainThread] logitech_receiver.base: (6) <= w[10 03 8100 000000]
20:18:21,445    DEBUG [MainThread] logitech_receiver.base: (6) => r[10 03 8100 110000]
     Notifications: keyboard multimedia raw, battery status (0x110000).
20:18:21,445    DEBUG [MainThread] logitech_receiver.base: (6) <= w[10 03 8101 000000]
20:18:21,453    DEBUG [MainThread] logitech_receiver.base: (6) => r[10 03 8101 420000]
     Features: special buttons, scroll accel (0x420000)
20:18:21,453    DEBUG [MainThread] logitech_receiver.base: (6) <= w[10 03 8107 000000]
20:18:21,461    DEBUG [MainThread] logitech_receiver.base: (6) => r[10 03 8107 070000]
     Battery: full, discharging.

20:18:21,461    DEBUG [MainThread] logitech_receiver.base: (4) <= w[10 FF 8102 000000]
20:18:21,468    DEBUG [MainThread] logitech_receiver.base: (4) => r[10 FF 8102 000200]
EX100 Receiver 27 Mhz
  Device path  : /dev/hidraw1
  USB id       : 046d:c517
  Serial       : None
20:18:21,468    DEBUG [MainThread] logitech_receiver.base: (4) <= w[10 FF 81F1 010000]
20:18:21,476    DEBUG [MainThread] logitech_receiver.base: (4) => r[10 FF 8F81 F10200]
20:18:21,476    DEBUG [MainThread] logitech_receiver.base: (4) device 0xFF error on request {81F1}: 2 = invalid address
  Has 2 paired device(s) out of a maximum of 4.
20:18:21,476    DEBUG [MainThread] logitech_receiver.base: (4) <= w[10 FF 8102 000000]
20:18:21,484    DEBUG [MainThread] logitech_receiver.base: (4) => r[10 FF 8102 000200]
20:18:21,484    DEBUG [MainThread] logitech_receiver.base: (4) <= w[10 FF 8100 000000]
20:18:21,492    DEBUG [MainThread] logitech_receiver.base: (4) => r[10 FF 8100 000100]
  Notifications: wireless (0x000100)
20:18:21,492    DEBUG [MainThread] logitech_receiver.base: (4) <= w[10 FF 83B3 000000]
20:18:21,500    DEBUG [MainThread] logitech_receiver.base: (4) => r[10 FF 8F83 B30100]
20:18:21,500    DEBUG [MainThread] logitech_receiver.base: (4) device 0xFF error on request {83B3}: 1 = invalid SubID/command
20:18:21,500    DEBUG [MainThread] logitech_receiver.base: (4) <= w[10 FF 8102 000000]
20:18:21,508    DEBUG [MainThread] logitech_receiver.base: (4) => r[10 FF 8102 000200]
20:18:21,508    DEBUG [MainThread] logitech_receiver.base: (4) <= w[10 FF 83B5 200000]
20:18:21,516    DEBUG [MainThread] logitech_receiver.base: (4) => r[10 FF 8F83 B50100]
20:18:21,516    DEBUG [MainThread] logitech_receiver.base: (4) device 0xFF error on request {83B5}: 1 = invalid SubID/command
20:18:21,521     INFO [MainThread] logitech_receiver.receiver: <EX100Receiver27Mhz(/dev/hidraw1,4)>: found new device 1 (3F00)

20:18:21,521    DEBUG [MainThread] logitech_receiver.base: (7) pinging device 1
20:18:21,521    DEBUG [MainThread] logitech_receiver.base: (7) <= w[10 01 001D 000092]
20:18:21,524    DEBUG [MainThread] logitech_receiver.base: (7) => r[10 01 8F00 1D0100]
  1: Wireless Mouse EX100
     Device path  : /dev/hidraw6
     USB id       : 046d:3F00
     Codename     : EX100m
     Kind         : mouse
     Protocol     : HID++ 1.0
20:18:21,524    DEBUG [MainThread] logitech_receiver.base: (4) <= w[10 FF 83B5 200000]
20:18:21,532    DEBUG [MainThread] logitech_receiver.base: (4) => r[10 FF 8F83 B50100]
20:18:21,532    DEBUG [MainThread] logitech_receiver.base: (4) device 0xFF error on request {83B5}: 1 = invalid SubID/command
20:18:21,532    DEBUG [MainThread] logitech_receiver.base: (4) <= w[10 FF 83B5 300000]
20:18:21,540    DEBUG [MainThread] logitech_receiver.base: (4) => r[10 FF 8F83 B50100]
20:18:21,540    DEBUG [MainThread] logitech_receiver.base: (4) device 0xFF error on request {83B5}: 1 = invalid SubID/command
20:18:21,540    DEBUG [MainThread] logitech_receiver.base: (4) <= w[10 FF 83D5 000000]
20:18:21,548    DEBUG [MainThread] logitech_receiver.base: (4) => r[10 FF 8F83 D50100]
20:18:21,548    DEBUG [MainThread] logitech_receiver.base: (4) device 0xFF error on request {83D5}: 1 = invalid SubID/command
     Serial number: ?
20:18:21,548    DEBUG [MainThread] logitech_receiver.base: (7) <= w[10 01 81F1 010000]
20:18:21,556    DEBUG [MainThread] logitech_receiver.base: (7) => r[10 01 8F81 F10200]
20:18:21,556    DEBUG [MainThread] logitech_receiver.base: (7) device 0x01 error on request {81F1}: 2 = invalid address
20:18:21,556    DEBUG [MainThread] logitech_receiver.base: (4) <= w[10 FF 83B5 300000]
20:18:21,564    DEBUG [MainThread] logitech_receiver.base: (4) => r[10 FF 8F83 B50100]
20:18:21,564    DEBUG [MainThread] logitech_receiver.base: (4) device 0xFF error on request {83B5}: 1 = invalid SubID/command
     The power switch is located on the (unknown).
20:18:21,564    DEBUG [MainThread] logitech_receiver.base: (7) <= w[10 01 8100 000000]
20:18:21,572    DEBUG [MainThread] logitech_receiver.base: (7) => r[10 01 8100 3C0000]
     Notifications: roller V, mouse extra buttons, battery status, roller H (0x3C0000).
20:18:21,572    DEBUG [MainThread] logitech_receiver.base: (7) <= w[10 01 8101 000000]
20:18:21,580    DEBUG [MainThread] logitech_receiver.base: (7) => r[10 01 8101 420000]
     Features: special buttons, scroll accel (0x420000)
20:18:21,580    DEBUG [MainThread] logitech_receiver.base: (7) <= w[10 01 8107 000000]
20:18:21,588    DEBUG [MainThread] logitech_receiver.base: (7) => r[10 01 8107 050000]
     Battery: good, discharging.
20:18:21,589    DEBUG [MainThread] logitech_receiver.base: (4) <= w[10 FF 83B5 210000]
20:18:21,596    DEBUG [MainThread] logitech_receiver.base: (4) => r[10 FF 8F83 B50100]
20:18:21,596    DEBUG [MainThread] logitech_receiver.base: (4) device 0xFF error on request {83B5}: 1 = invalid SubID/command
20:18:21,596    ERROR [MainThread] logitech_receiver.device: failed to set fake EX100 wpid for device 2 of <EX100Receiver27Mhz(/dev/hidraw1,4)>
20:18:21,596    ERROR [MainThread] logitech_receiver.receiver: register_new_device
Traceback (most recent call last):
  File "/usr/share/solaar/lib/logitech_receiver/receiver.py", line 150, in register_new_device
    dev = Device(self, number, notification)
  File "/usr/share/solaar/lib/logitech_receiver/device.py", line 116, in __init__
    raise _base.NoSuchDevice(number=number, receiver=receiver, error='Unknown EX100 device')
logitech_receiver.base.NoSuchDevice: {'number': 2, 'receiver': <EX100Receiver27Mhz(/dev/hidraw1,4)>, 'error': 'Unknown EX100 device'}
20:18:21,597  WARNING [MainThread] logitech_receiver.receiver: <EX100Receiver27Mhz(/dev/hidraw1,4)>: looked for device 2, not found
20:18:21,597    DEBUG [MainThread] logitech_receiver.base: (4) <= w[10 FF 83B5 220000]
20:18:21,604    DEBUG [MainThread] logitech_receiver.base: (4) => r[10 FF 8F83 B50100]
20:18:21,604    DEBUG [MainThread] logitech_receiver.base: (4) device 0xFF error on request {83B5}: 1 = invalid SubID/command
20:18:21,608     INFO [MainThread] logitech_receiver.receiver: <EX100Receiver27Mhz(/dev/hidraw1,4)>: found new device 3 (6500)

20:18:21,608    DEBUG [MainThread] logitech_receiver.base: (8) pinging device 3
20:18:21,608    DEBUG [MainThread] logitech_receiver.base: (8) <= w[10 03 0018 0000F9]
20:18:21,612    DEBUG [MainThread] logitech_receiver.base: (8) => r[10 03 8F00 180100]
  3: Wireless Keyboard EX100
     Device path  : /dev/hidraw4
     USB id       : 046d:6500
     Codename     : EX100
     Kind         : keyboard
     Protocol     : HID++ 1.0
20:18:21,612    DEBUG [MainThread] logitech_receiver.base: (4) <= w[10 FF 83B5 220000]
20:18:21,620    DEBUG [MainThread] logitech_receiver.base: (4) => r[10 FF 8F83 B50100]
20:18:21,620    DEBUG [MainThread] logitech_receiver.base: (4) device 0xFF error on request {83B5}: 1 = invalid SubID/command
20:18:21,620    DEBUG [MainThread] logitech_receiver.base: (4) <= w[10 FF 83B5 320000]
20:18:21,628    DEBUG [MainThread] logitech_receiver.base: (4) => r[10 FF 8F83 B50100]
20:18:21,628    DEBUG [MainThread] logitech_receiver.base: (4) device 0xFF error on request {83B5}: 1 = invalid SubID/command
20:18:21,628    DEBUG [MainThread] logitech_receiver.base: (4) <= w[10 FF 83D5 000000]
20:18:21,636    DEBUG [MainThread] logitech_receiver.base: (4) => r[10 FF 8F83 D50100]
20:18:21,636    DEBUG [MainThread] logitech_receiver.base: (4) device 0xFF error on request {83D5}: 1 = invalid SubID/command
     Serial number: ?
20:18:21,636    DEBUG [MainThread] logitech_receiver.base: (8) <= w[10 03 81F1 010000]
20:18:21,644    DEBUG [MainThread] logitech_receiver.base: (8) => r[10 03 8F81 F10200]
20:18:21,644    DEBUG [MainThread] logitech_receiver.base: (8) device 0x03 error on request {81F1}: 2 = invalid address
20:18:21,644    DEBUG [MainThread] logitech_receiver.base: (4) <= w[10 FF 83B5 320000]
20:18:21,652    DEBUG [MainThread] logitech_receiver.base: (4) => r[10 FF 8F83 B50100]
20:18:21,652    DEBUG [MainThread] logitech_receiver.base: (4) device 0xFF error on request {83B5}: 1 = invalid SubID/command
     The power switch is located on the (unknown).
20:18:21,652    DEBUG [MainThread] logitech_receiver.base: (8) <= w[10 03 8100 000000]
20:18:21,660    DEBUG [MainThread] logitech_receiver.base: (8) => r[10 03 8100 110000]
     Notifications: keyboard multimedia raw, battery status (0x110000).
20:18:21,660    DEBUG [MainThread] logitech_receiver.base: (8) <= w[10 03 8101 000000]
20:18:21,668    DEBUG [MainThread] logitech_receiver.base: (8) => r[10 03 8101 420000]
     Features: special buttons, scroll accel (0x420000)
20:18:21,668    DEBUG [MainThread] logitech_receiver.base: (8) <= w[10 03 8107 000000]
20:18:21,676    DEBUG [MainThread] logitech_receiver.base: (8) => r[10 03 8107 050000]
     Battery: good, discharging.

```

</details>

<p>
</details>

<details>
<summary>solaar -dd</summary>

```
20:21:12,279     INFO [MainThread] root: language hr_HR (UTF-8), translations path None
20:21:12,364    DEBUG [MainThread] solaar.ui.tray: using AppIndicator3
20:21:12,375     INFO [MainThread] solaar.upower: connected to system dbus, watching for suspend/resume events
20:21:12,419    DEBUG [MainThread] solaar.ui: startup registered=True, remote=False
20:21:12,420    DEBUG [AsyncUI] solaar.tasks: started
20:21:12,420     INFO [MainThread] solaar.ui.notify: starting desktop notifications
20:21:12,426    DEBUG [MainThread] solaar.ui.icons: sys.path[0] = /usr/share/solaar/lib
20:21:12,426    DEBUG [MainThread] solaar.ui.icons: looking for icons in /usr/share/solaar/icons
20:21:12,426    DEBUG [MainThread] solaar.ui.icons: looking for icons in /usr/share/solaar/share/solaar/icons
20:21:12,426    DEBUG [MainThread] solaar.ui.icons: looking for icons in /home/goran/.local/share/solaar/icons
20:21:12,426    DEBUG [MainThread] solaar.ui.icons: looking for icons in /usr/share/solaar/lib/share/solaar/icons
20:21:12,426    DEBUG [MainThread] solaar.ui.icons: looking for icons in /usr/share/solaar/share/solaar/icons
20:21:12,426    DEBUG [MainThread] solaar.ui.icons: looking for icons in /home/goran/.local/share/flatpak/exports/share/solaar/icons
20:21:12,426    DEBUG [MainThread] solaar.ui.icons: looking for icons in /var/lib/flatpak/exports/share/solaar/icons
20:21:12,426    DEBUG [MainThread] solaar.ui.icons: looking for icons in /usr/share/cinnamon/solaar/icons
20:21:12,426    DEBUG [MainThread] solaar.ui.icons: looking for icons in /usr/share/gnome/solaar/icons
20:21:12,426    DEBUG [MainThread] solaar.ui.icons: looking for icons in /home/goran/.local/share/flatpak/exports/share/solaar/icons
20:21:12,427    DEBUG [MainThread] solaar.ui.icons: looking for icons in /var/lib/flatpak/exports/share/solaar/icons
20:21:12,427    DEBUG [MainThread] solaar.ui.icons: looking for icons in /usr/local/share/solaar/icons
20:21:12,427    DEBUG [MainThread] solaar.ui.icons: looking for icons in /usr/share/solaar/icons
20:21:12,427    DEBUG [MainThread] solaar.ui.icons: looking for icons in /var/lib/snapd/desktop/solaar/icons
20:21:12,427    DEBUG [MainThread] solaar.ui.icons: icon theme paths: ['/usr/share/solaar/icons', '/usr/share/solaar/icons', '/home/goran/.local/share/icons', '/home/goran/.icons', '/home/goran/.local/share/flatpak/exports/share/icons', '/var/lib/flatpak/exports/share/icons', '/usr/share/cinnamon/icons', '/usr/share/gnome/icons', '/home/goran/.local/share/flatpak/exports/share/icons', '/var/lib/flatpak/exports/share/icons', '/usr/local/share/icons', '/usr/share/icons', '/var/lib/snapd/desktop/icons', '/home/goran/.local/share/flatpak/exports/share/pixmaps', '/var/lib/flatpak/exports/share/pixmaps', '/usr/share/cinnamon/pixmaps', '/usr/share/gnome/pixmaps', '/home/goran/.local/share/flatpak/exports/share/pixmaps', '/var/lib/flatpak/exports/share/pixmaps', '/usr/local/share/pixmaps', '/usr/share/pixmaps', '/var/lib/snapd/desktop/pixmaps']
20:21:12,497     INFO [MainThread] solaar.listener: starting receiver listening threads
20:21:12,525     INFO [MainThread] solaar.listener: receiver event add DeviceInfo(path='/dev/hidraw3', vendor_id='046d', product_id='c517', serial='', release=b'3810', manufacturer=b'Logitech', product=b'USB Receiver', interface=1, driver='logitech-djreceiver')
20:21:12,526    DEBUG [MainThread] logitech_receiver.base: (14) <= w[10 FF 83B5 030000]
20:21:12,535    DEBUG [MainThread] logitech_receiver.base: (14) => r[10 FF 8F83 B50100]
20:21:12,535    DEBUG [MainThread] logitech_receiver.base: (14) device 0xFF error on request {83B5}: 1 = invalid SubID/command
20:21:12,535     INFO [ReceiverListener:hidraw3] logitech_receiver.listener: started with <EX100Receiver27Mhz(/dev/hidraw3,14)> (14)
20:21:12,535     INFO [ReceiverListener:hidraw3] solaar.listener: <EX100Receiver27Mhz(/dev/hidraw3,14)>: notifications listener has started (14)
20:21:12,536    DEBUG [ReceiverListener:hidraw3] logitech_receiver.base: (14) <= w[10 FF 8000 100900]
20:21:12,539     INFO [MainThread] solaar.listener: receiver event add DeviceInfo(path='/dev/hidraw1', vendor_id='046d', product_id='c517', serial='', release=b'3820', manufacturer=b'Logitech', product=b'USB Receiver', interface=1, driver='logitech-djreceiver')
20:21:12,539    DEBUG [MainThread] logitech_receiver.base: (15) <= w[10 FF 83B5 030000]
20:21:12,543    DEBUG [ReceiverListener:hidraw3] logitech_receiver.base: (14) => r[10 FF 8000 000000]
20:21:12,543    DEBUG [ReceiverListener:hidraw3] logitech_receiver.base: (14) <= w[10 FF 8100 000000]
20:21:12,544    DEBUG [MainThread] logitech_receiver.base: (15) => r[10 FF 8F83 B50100]
20:21:12,544    DEBUG [MainThread] logitech_receiver.base: (15) device 0xFF error on request {83B5}: 1 = invalid SubID/command
20:21:12,544     INFO [ReceiverListener:hidraw1] logitech_receiver.listener: started with <EX100Receiver27Mhz(/dev/hidraw1,15)> (15)
20:21:12,544     INFO [ReceiverListener:hidraw1] solaar.listener: <EX100Receiver27Mhz(/dev/hidraw1,15)>: notifications listener has started (15)
20:21:12,545    DEBUG [ReceiverListener:hidraw1] logitech_receiver.base: (15) <= w[10 FF 8000 100900]
20:21:12,551    DEBUG [ReceiverListener:hidraw3] logitech_receiver.base: (14) => r[10 FF 8100 000100]
20:21:12,551     INFO [ReceiverListener:hidraw3] logitech_receiver.receiver: <EX100Receiver27Mhz(/dev/hidraw3,14)>: receiver notifications enabled => ('wireless',)
20:21:12,551    DEBUG [ReceiverListener:hidraw3] logitech_receiver.base: (14) <= w[10 FF 8002 020000]
20:21:12,552    DEBUG [ReceiverListener:hidraw1] logitech_receiver.base: (15) => r[10 FF 8000 000000]
20:21:12,552    DEBUG [ReceiverListener:hidraw1] logitech_receiver.base: (15) <= w[10 FF 8100 000000]
20:21:12,559    DEBUG [ReceiverListener:hidraw3] logitech_receiver.base: (14) => r[10 FF 8002 000000]
20:21:12,559     INFO [ReceiverListener:hidraw3] solaar.listener: status_changed <EX100Receiver27Mhz(/dev/hidraw3,14)>: present, Nema uparenih uređaja. (0) 
20:21:12,560    DEBUG [ReceiverListener:hidraw1] logitech_receiver.base: (15) => r[10 FF 8100 000100]
20:21:12,560     INFO [ReceiverListener:hidraw1] logitech_receiver.receiver: <EX100Receiver27Mhz(/dev/hidraw1,15)>: receiver notifications enabled => ('wireless',)
20:21:12,560    DEBUG [ReceiverListener:hidraw1] logitech_receiver.base: (15) <= w[10 FF 8002 020000]
20:21:12,567    DEBUG [ReceiverListener:hidraw3] logitech_receiver.base: (14) => r[10 01 4102 012236]
20:21:12,568    DEBUG [ReceiverListener:hidraw1] logitech_receiver.base: (15) => r[10 FF 8002 000000]
20:21:12,568     INFO [ReceiverListener:hidraw1] solaar.listener: status_changed <EX100Receiver27Mhz(/dev/hidraw1,15)>: present, Nema uparenih uređaja. (0) 
20:21:12,571     INFO [ReceiverListener:hidraw3] logitech_receiver.receiver: <EX100Receiver27Mhz(/dev/hidraw3,14)>: found new device 1 (3622)
20:21:12,571    DEBUG [ReceiverListener:hidraw3] logitech_receiver.base: (14) <= w[10 FF 83B5 300000]
20:21:12,575    DEBUG [ReceiverListener:hidraw3] logitech_receiver.base: (14) => r[10 03 4102 021256]
20:21:12,584    DEBUG [ReceiverListener:hidraw3] logitech_receiver.base: (14) => r[10 FF 8F83 B50100]
20:21:12,584    DEBUG [ReceiverListener:hidraw3] logitech_receiver.base: (14) device 0xFF error on request {83B5}: 1 = invalid SubID/command
20:21:12,584    DEBUG [ReceiverListener:hidraw3] logitech_receiver.base: (14) <= w[10 FF 83D5 000000]
20:21:12,591    DEBUG [ReceiverListener:hidraw3] logitech_receiver.base: (14) => r[10 03 8100 110000]
20:21:12,599    DEBUG [ReceiverListener:hidraw3] logitech_receiver.base: (14) => r[10 FF 8F83 D50100]
20:21:12,599    DEBUG [ReceiverListener:hidraw3] logitech_receiver.base: (14) device 0xFF error on request {83D5}: 1 = invalid SubID/command
20:21:12,571     INFO [ReceiverListener:hidraw3] solaar.listener: connection Notification(1,41,02,012236) for <Device(1,3622,Wireless Keyboard S510,?)> (keyboard)
20:21:12,576    DEBUG [ReceiverListener:hidraw1] logitech_receiver.base: (15) => r[10 01 4102 012185]
20:21:12,600    DEBUG [ReceiverListener:hidraw3] solaar.configuration: load => {'102A:42FBBA88': {'_name': 'G700s Gaming Mouse', 'side-scroll': True, 'smooth-scroll': False}, '3600:?': {'_name': '? (3600)'}, '3622:?': {'_name': 'Wireless Keyboard S510'}, '5600:?': {'_name': '? (5600)'}, '5612:?': {'_name': 'LX5 Cordless Mouse'}, '6800:?': {'_name': '? (6800)'}, '6822:?': {'_name': 'Wireless Mouse M30'}, '8500:?': {'_name': '? (8500)'}, '8521:?': {'_name': 'Wireless Keyboard MK300'}, '8522:?': {'_name': 'Wireless Keyboard S510'}, '_version': '1.0.3'}
20:21:12,601    DEBUG [ReceiverListener:hidraw3] logitech_receiver.base: (14) <= w[10 FF 83B5 300000]
20:21:12,605     INFO [ReceiverListener:hidraw1] logitech_receiver.receiver: <EX100Receiver27Mhz(/dev/hidraw1,15)>: found new device 1 (8521)
20:21:12,605    DEBUG [ReceiverListener:hidraw1] logitech_receiver.base: (15) <= w[10 FF 83B5 300000]
20:21:12,608    DEBUG [ReceiverListener:hidraw1] logitech_receiver.base: (15) => r[10 01 8000 000000]
20:21:12,616    DEBUG [ReceiverListener:hidraw1] logitech_receiver.base: (15) => r[10 FF 8F83 B50100]
20:21:12,616    DEBUG [ReceiverListener:hidraw1] logitech_receiver.base: (15) device 0xFF error on request {83B5}: 1 = invalid SubID/command
20:21:12,616    DEBUG [ReceiverListener:hidraw1] logitech_receiver.base: (15) <= w[10 FF 83D5 000000]
20:21:12,624    DEBUG [ReceiverListener:hidraw1] logitech_receiver.base: (15) => r[10 01 8100 3C0000]
20:21:12,632    DEBUG [ReceiverListener:hidraw1] logitech_receiver.base: (15) => r[10 FF 8F83 D50100]
20:21:12,632    DEBUG [ReceiverListener:hidraw1] logitech_receiver.base: (15) device 0xFF error on request {83D5}: 1 = invalid SubID/command
20:21:12,605     INFO [ReceiverListener:hidraw1] solaar.listener: connection Notification(1,41,02,012185) for <Device(1,8521,Wireless Keyboard MK300,?)> (keyboard)
20:21:12,607    DEBUG [ReceiverListener:hidraw3] logitech_receiver.base: (14) => r[10 03 8000 000000]
20:21:12,632    DEBUG [ReceiverListener:hidraw1] logitech_receiver.base: (15) <= w[10 FF 83B5 300000]
20:21:12,632    DEBUG [ReceiverListener:hidraw3] logitech_receiver.base: (14) => r[10 FF 8F83 B50100]
20:21:12,632    DEBUG [ReceiverListener:hidraw3] logitech_receiver.base: (14) device 0xFF error on request {83B5}: 1 = invalid SubID/command
20:21:12,633    DEBUG [ReceiverListener:hidraw3] logitech_receiver.base: (14) <= w[10 FF 83D5 000000]
20:21:12,639    DEBUG [ReceiverListener:hidraw3] logitech_receiver.base: (14) => r[10 03 8107 070000]
20:21:12,640    DEBUG [ReceiverListener:hidraw1] logitech_receiver.base: (15) => r[10 01 8000 000000]
20:21:12,647    DEBUG [ReceiverListener:hidraw3] logitech_receiver.base: (14) => r[10 FF 8F83 D50100]
20:21:12,647    DEBUG [ReceiverListener:hidraw3] logitech_receiver.base: (14) device 0xFF error on request {83D5}: 1 = invalid SubID/command
20:21:12,647     INFO [ReceiverListener:hidraw3] solaar.listener: status_changed <EX100Receiver27Mhz(/dev/hidraw3,14)>: present, 1 upareni uređaj. (0) 
20:21:12,648    DEBUG [ReceiverListener:hidraw1] logitech_receiver.base: (15) => r[10 FF 8F83 B50100]
20:21:12,648    DEBUG [ReceiverListener:hidraw1] logitech_receiver.base: (15) device 0xFF error on request {83B5}: 1 = invalid SubID/command
20:21:12,648    DEBUG [ReceiverListener:hidraw1] logitech_receiver.base: (15) <= w[10 FF 83D5 000000]
20:21:12,648    DEBUG [ReceiverListener:hidraw3] logitech_receiver.base: (14) <= w[10 FF 83B5 300000]
20:21:12,655    DEBUG [ReceiverListener:hidraw3] logitech_receiver.base: (14) => r[10 FF 8F83 B50100]
20:21:12,655    DEBUG [ReceiverListener:hidraw3] logitech_receiver.base: (14) device 0xFF error on request {83B5}: 1 = invalid SubID/command
20:21:12,655    DEBUG [ReceiverListener:hidraw3] logitech_receiver.base: (14) <= w[10 FF 83D5 000000]
20:21:12,656    DEBUG [ReceiverListener:hidraw1] logitech_receiver.base: (15) => r[10 01 8100 3C0000]
20:21:12,663    DEBUG [ReceiverListener:hidraw3] logitech_receiver.base: (14) => r[10 FF 8F83 D50100]
20:21:12,663    DEBUG [ReceiverListener:hidraw3] logitech_receiver.base: (14) device 0xFF error on request {83D5}: 1 = invalid SubID/command
20:21:12,663    ERROR [ReceiverListener:hidraw3] logitech_receiver.listener: processing Notification(1,41,02,012236)
Traceback (most recent call last):
  File "/usr/share/solaar/lib/logitech_receiver/listener.py", line 193, in run
    self._notifications_callback(n)
  File "/usr/share/solaar/lib/solaar/listener.py", line 245, in _notifications_handler
    _notifications.process(dev, n)
  File "/usr/share/solaar/lib/logitech_receiver/notifications.py", line 61, in process
    return _process_device_notification(device, status, notification)
  File "/usr/share/solaar/lib/logitech_receiver/notifications.py", line 111, in _process_device_notification
    return _process_hidpp10_notification(device, status, n)
  File "/usr/share/solaar/lib/logitech_receiver/notifications.py", line 210, in _process_hidpp10_notification
    assert wpid == device.wpid, '%s wpid mismatch, got %s' % (device, wpid)
AssertionError: <Device(1,3622,Wireless Keyboard S510,?)> wpid mismatch, got 3600
20:21:12,664    DEBUG [ReceiverListener:hidraw1] logitech_receiver.base: (15) => r[10 FF 8F83 D50100]
20:21:12,665    DEBUG [ReceiverListener:hidraw1] logitech_receiver.base: (15) device 0xFF error on request {83D5}: 1 = invalid SubID/command
20:21:12,665     INFO [ReceiverListener:hidraw1] solaar.listener: status_changed <EX100Receiver27Mhz(/dev/hidraw1,15)>: present, 1 upareni uređaj. (0) 
20:21:12,667    DEBUG [ReceiverListener:hidraw1] logitech_receiver.base: (15) <= w[10 FF 83B5 300000]
20:21:12,669     INFO [ReceiverListener:hidraw3] logitech_receiver.receiver: <EX100Receiver27Mhz(/dev/hidraw3,14)>: found new device 3 (5612)
20:21:12,669    DEBUG [ReceiverListener:hidraw3] logitech_receiver.base: (14) <= w[10 FF 83B5 320000]
20:21:12,671    DEBUG [ReceiverListener:hidraw3] logitech_receiver.base: (14) => r[10 FF 8F83 B50100]
20:21:12,671    DEBUG [ReceiverListener:hidraw3] logitech_receiver.base: (14) device 0xFF error on request {83B5}: 1 = invalid SubID/command
20:21:12,671    DEBUG [ReceiverListener:hidraw3] logitech_receiver.base: (14) <= w[10 FF 83D5 000000]
20:21:12,679    DEBUG [ReceiverListener:hidraw3] logitech_receiver.base: (14) => r[10 FF 8F83 D50100]
20:21:12,679    DEBUG [ReceiverListener:hidraw3] logitech_receiver.base: (14) device 0xFF error on request {83D5}: 1 = invalid SubID/command
20:21:12,669     INFO [ReceiverListener:hidraw3] solaar.listener: connection Notification(3,41,02,021256) for <Device(3,5612,LX5 Cordless Mouse,?)> (mouse)
20:21:12,670    DEBUG [MainThread] solaar.ui: activate
20:21:12,672    DEBUG [ReceiverListener:hidraw1] logitech_receiver.base: (15) => r[10 01 8000 000000]
20:21:12,679    DEBUG [ReceiverListener:hidraw3] logitech_receiver.base: (14) <= w[10 FF 83B5 320000]
20:21:12,680    DEBUG [ReceiverListener:hidraw1] logitech_receiver.base: (15) => r[10 FF 8F83 B50100]
20:21:12,680    DEBUG [ReceiverListener:hidraw1] logitech_receiver.base: (15) device 0xFF error on request {83B5}: 1 = invalid SubID/command
20:21:12,680    DEBUG [ReceiverListener:hidraw1] logitech_receiver.base: (15) <= w[10 FF 83D5 000000]
20:21:12,682    DEBUG [MainThread] solaar.ui: status changed: <EX100Receiver27Mhz(/dev/hidraw3,14)> (NONE) None
20:21:12,684    DEBUG [MainThread] solaar.ui.window: new receiver row ('/dev/hidraw3', 0, True, 'EX100 Receiver 27 Mhz', 'preferences-desktop-peripherals', None, None, <EX100Receiver27Mhz(/dev/hidraw3,14)>)
20:21:12,684    DEBUG [MainThread] solaar.ui: status changed: <EX100Receiver27Mhz(/dev/hidraw1,15)> (NONE) None
20:21:12,685    DEBUG [MainThread] solaar.ui.window: new receiver row ('/dev/hidraw1', 0, True, 'EX100 Receiver 27 Mhz', 'preferences-desktop-peripherals', None, None, <EX100Receiver27Mhz(/dev/hidraw1,15)>)
20:21:12,685    DEBUG [MainThread] solaar.ui: status changed: <EX100Receiver27Mhz(/dev/hidraw3,14)> (NONE) None
20:21:12,685    DEBUG [MainThread] solaar.ui: status changed: <EX100Receiver27Mhz(/dev/hidraw1,15)> (NONE) None
20:21:12,687    DEBUG [ReceiverListener:hidraw3] logitech_receiver.base: (14) => r[10 FF 8F83 B50100]
20:21:12,687    DEBUG [ReceiverListener:hidraw3] logitech_receiver.base: (14) device 0xFF error on request {83B5}: 1 = invalid SubID/command
20:21:12,687    DEBUG [ReceiverListener:hidraw3] logitech_receiver.base: (14) <= w[10 FF 83D5 000000]
20:21:12,688    DEBUG [ReceiverListener:hidraw1] logitech_receiver.base: (15) => r[10 01 8107 050000]
20:21:12,695    DEBUG [ReceiverListener:hidraw3] logitech_receiver.base: (14) => r[10 FF 8F83 D50100]
20:21:12,695    DEBUG [ReceiverListener:hidraw3] logitech_receiver.base: (14) device 0xFF error on request {83D5}: 1 = invalid SubID/command
20:21:12,695     INFO [ReceiverListener:hidraw3] solaar.listener: status_changed <EX100Receiver27Mhz(/dev/hidraw3,14)>: present, 2 uparena uređaja. (0) 
20:21:12,696    DEBUG [ReceiverListener:hidraw3] logitech_receiver.base: (14) <= w[10 FF 83B5 320000]
20:21:12,696    DEBUG [ReceiverListener:hidraw1] logitech_receiver.base: (15) => r[10 FF 8F83 D50100]
20:21:12,696    DEBUG [ReceiverListener:hidraw1] logitech_receiver.base: (15) device 0xFF error on request {83D5}: 1 = invalid SubID/command
20:21:12,696    ERROR [ReceiverListener:hidraw1] logitech_receiver.listener: processing Notification(1,41,02,012185)
Traceback (most recent call last):
  File "/usr/share/solaar/lib/logitech_receiver/listener.py", line 193, in run
    self._notifications_callback(n)
  File "/usr/share/solaar/lib/solaar/listener.py", line 245, in _notifications_handler
    _notifications.process(dev, n)
  File "/usr/share/solaar/lib/logitech_receiver/notifications.py", line 61, in process
    return _process_device_notification(device, status, notification)
  File "/usr/share/solaar/lib/logitech_receiver/notifications.py", line 111, in _process_device_notification
    return _process_hidpp10_notification(device, status, n)
  File "/usr/share/solaar/lib/logitech_receiver/notifications.py", line 210, in _process_hidpp10_notification
    assert wpid == device.wpid, '%s wpid mismatch, got %s' % (device, wpid)
AssertionError: <Device(1,8521,Wireless Keyboard MK300,?)> wpid mismatch, got 8500
20:21:12,701     INFO [ReceiverListener:hidraw1] logitech_receiver.receiver: <EX100Receiver27Mhz(/dev/hidraw1,15)>: found new device 3 (6822)
20:21:12,701    DEBUG [ReceiverListener:hidraw1] logitech_receiver.base: (15) <= w[10 FF 83B5 320000]
20:21:12,704    DEBUG [ReceiverListener:hidraw1] logitech_receiver.base: (15) => r[10 FF 8F83 B50100]
20:21:12,704    DEBUG [ReceiverListener:hidraw1] logitech_receiver.base: (15) device 0xFF error on request {83B5}: 1 = invalid SubID/command
20:21:12,704    DEBUG [ReceiverListener:hidraw1] logitech_receiver.base: (15) <= w[10 FF 83D5 000000]
20:21:12,712    DEBUG [ReceiverListener:hidraw1] logitech_receiver.base: (15) => r[10 FF 8F83 D50100]
20:21:12,712    DEBUG [ReceiverListener:hidraw1] logitech_receiver.base: (15) device 0xFF error on request {83D5}: 1 = invalid SubID/command
20:21:12,701     INFO [ReceiverListener:hidraw1] solaar.listener: connection Notification(3,41,02,022268) for <Device(3,6822,Wireless Mouse M30,?)> (mouse)
20:21:12,701    DEBUG [MainThread] solaar.ui: status changed: <EX100Receiver27Mhz(/dev/hidraw3,14)> (NONE) None
20:21:12,703    DEBUG [ReceiverListener:hidraw3] logitech_receiver.base: (14) => r[10 FF 8F83 B50100]
20:21:12,712    DEBUG [ReceiverListener:hidraw1] logitech_receiver.base: (15) <= w[10 FF 83B5 320000]
20:21:12,712    DEBUG [ReceiverListener:hidraw3] logitech_receiver.base: (14) device 0xFF error on request {83B5}: 1 = invalid SubID/command
20:21:12,713    DEBUG [ReceiverListener:hidraw3] logitech_receiver.base: (14) <= w[10 FF 83D5 000000]
20:21:12,719    DEBUG [ReceiverListener:hidraw3] logitech_receiver.base: (14) => r[10 FF 8F83 D50100]
20:21:12,719    DEBUG [ReceiverListener:hidraw3] logitech_receiver.base: (14) device 0xFF error on request {83D5}: 1 = invalid SubID/command
20:21:12,719    ERROR [ReceiverListener:hidraw3] logitech_receiver.listener: processing Notification(3,41,02,021256)
Traceback (most recent call last):
  File "/usr/share/solaar/lib/logitech_receiver/listener.py", line 193, in run
    self._notifications_callback(n)
  File "/usr/share/solaar/lib/solaar/listener.py", line 245, in _notifications_handler
    _notifications.process(dev, n)
  File "/usr/share/solaar/lib/logitech_receiver/notifications.py", line 61, in process
    return _process_device_notification(device, status, notification)
  File "/usr/share/solaar/lib/logitech_receiver/notifications.py", line 111, in _process_device_notification
    return _process_hidpp10_notification(device, status, n)
  File "/usr/share/solaar/lib/logitech_receiver/notifications.py", line 210, in _process_hidpp10_notification
    assert wpid == device.wpid, '%s wpid mismatch, got %s' % (device, wpid)
AssertionError: <Device(3,5612,LX5 Cordless Mouse,?)> wpid mismatch, got 5600
20:21:12,720    DEBUG [ReceiverListener:hidraw1] logitech_receiver.base: (15) => r[10 FF 8F83 B50100]
20:21:12,720    DEBUG [ReceiverListener:hidraw1] logitech_receiver.base: (15) device 0xFF error on request {83B5}: 1 = invalid SubID/command
20:21:12,720    DEBUG [ReceiverListener:hidraw1] logitech_receiver.base: (15) <= w[10 FF 83D5 000000]
20:21:12,728    DEBUG [ReceiverListener:hidraw1] logitech_receiver.base: (15) => r[10 FF 8F83 D50100]
20:21:12,728    DEBUG [ReceiverListener:hidraw1] logitech_receiver.base: (15) device 0xFF error on request {83D5}: 1 = invalid SubID/command
20:21:12,728     INFO [ReceiverListener:hidraw1] solaar.listener: status_changed <EX100Receiver27Mhz(/dev/hidraw1,15)>: present, 2 uparena uređaja. (0) 
20:21:12,728    DEBUG [MainThread] solaar.ui: status changed: <EX100Receiver27Mhz(/dev/hidraw1,15)> (NONE) None
20:21:12,728    DEBUG [ReceiverListener:hidraw1] logitech_receiver.base: (15) <= w[10 FF 83B5 320000]
20:21:12,736    DEBUG [ReceiverListener:hidraw1] logitech_receiver.base: (15) => r[10 FF 8F83 B50100]
20:21:12,736    DEBUG [ReceiverListener:hidraw1] logitech_receiver.base: (15) device 0xFF error on request {83B5}: 1 = invalid SubID/command
20:21:12,736    DEBUG [ReceiverListener:hidraw1] logitech_receiver.base: (15) <= w[10 FF 83D5 000000]
20:21:12,744    DEBUG [ReceiverListener:hidraw1] logitech_receiver.base: (15) => r[10 FF 8F83 D50100]
20:21:12,744    DEBUG [ReceiverListener:hidraw1] logitech_receiver.base: (15) device 0xFF error on request {83D5}: 1 = invalid SubID/command
20:21:12,744    ERROR [ReceiverListener:hidraw1] logitech_receiver.listener: processing Notification(3,41,02,022268)
Traceback (most recent call last):
  File "/usr/share/solaar/lib/logitech_receiver/listener.py", line 193, in run
    self._notifications_callback(n)
  File "/usr/share/solaar/lib/solaar/listener.py", line 245, in _notifications_handler
    _notifications.process(dev, n)
  File "/usr/share/solaar/lib/logitech_receiver/notifications.py", line 61, in process
    return _process_device_notification(device, status, notification)
  File "/usr/share/solaar/lib/logitech_receiver/notifications.py", line 111, in _process_device_notification
    return _process_hidpp10_notification(device, status, n)
  File "/usr/share/solaar/lib/logitech_receiver/notifications.py", line 210, in _process_hidpp10_notification
    assert wpid == device.wpid, '%s wpid mismatch, got %s' % (device, wpid)
AssertionError: <Device(3,6822,Wireless Mouse M30,?)> wpid mismatch, got 6800
20:21:16,161    DEBUG [ReceiverListener:hidraw1] logitech_receiver.base: (15) => r[10 01 0501 000000]
20:21:16,177    DEBUG [ReceiverListener:hidraw1] logitech_receiver.base: (15) => r[10 01 0501 000000]
20:21:16,185    DEBUG [ReceiverListener:hidraw1] logitech_receiver.base: (15) => r[10 01 0501 000000]
20:21:16,209    DEBUG [ReceiverListener:hidraw1] logitech_receiver.base: (15) => r[10 01 0501 000000]
20:21:16,449    DEBUG [ReceiverListener:hidraw1] logitech_receiver.base: (15) => r[10 01 0502 000000]
20:21:16,465    DEBUG [ReceiverListener:hidraw1] logitech_receiver.base: (15) => r[10 01 0501 000000]
20:21:16,489    DEBUG [ReceiverListener:hidraw1] logitech_receiver.base: (15) => r[10 01 0501 000000]
20:21:16,705    DEBUG [ReceiverListener:hidraw1] logitech_receiver.base: (15) => r[10 01 0502 000000]
20:21:16,729    DEBUG [ReceiverListener:hidraw1] logitech_receiver.base: (15) => r[10 01 0501 000000]
20:21:16,737    DEBUG [ReceiverListener:hidraw1] logitech_receiver.base: (15) => r[10 01 0501 000000]
20:21:16,929    DEBUG [ReceiverListener:hidraw1] logitech_receiver.base: (15) => r[10 01 0502 000000]
20:21:16,945    DEBUG [ReceiverListener:hidraw1] logitech_receiver.base: (15) => r[10 01 0501 000000]
20:21:16,953    DEBUG [ReceiverListener:hidraw1] logitech_receiver.base: (15) => r[10 01 0501 000000]
20:21:16,993    DEBUG [ReceiverListener:hidraw1] logitech_receiver.base: (15) => r[10 01 0501 000000]
20:21:17,145    DEBUG [ReceiverListener:hidraw1] logitech_receiver.base: (15) => r[10 01 0501 000000]
20:21:17,153    DEBUG [ReceiverListener:hidraw1] logitech_receiver.base: (15) => r[10 01 0501 000000]
20:21:17,169    DEBUG [ReceiverListener:hidraw1] logitech_receiver.base: (15) => r[10 01 0501 000000]
20:21:17,193    DEBUG [ReceiverListener:hidraw1] logitech_receiver.base: (15) => r[10 01 0501 000000]
20:21:17,409    DEBUG [ReceiverListener:hidraw1] logitech_receiver.base: (15) => r[10 01 0502 000000]
20:21:17,433    DEBUG [ReceiverListener:hidraw1] logitech_receiver.base: (15) => r[10 01 0501 000000]
20:21:17,449    DEBUG [ReceiverListener:hidraw1] logitech_receiver.base: (15) => r[10 01 0501 000000]
20:21:17,642    DEBUG [ReceiverListener:hidraw1] logitech_receiver.base: (15) => r[10 01 0502 000000]
20:21:17,657    DEBUG [ReceiverListener:hidraw1] logitech_receiver.base: (15) => r[10 01 0501 000000]
20:21:17,689    DEBUG [ReceiverListener:hidraw1] logitech_receiver.base: (15) => r[10 01 0501 000000]
20:21:17,897    DEBUG [ReceiverListener:hidraw1] logitech_receiver.base: (15) => r[10 01 0502 000000]
20:21:17,913    DEBUG [ReceiverListener:hidraw1] logitech_receiver.base: (15) => r[10 01 0501 000000]
20:21:17,929    DEBUG [ReceiverListener:hidraw1] logitech_receiver.base: (15) => r[10 01 0501 000000]
20:21:19,498    DEBUG [ReceiverListener:hidraw1] logitech_receiver.base: (15) => r[10 01 05FF 000000]
20:21:19,522    DEBUG [ReceiverListener:hidraw1] logitech_receiver.base: (15) => r[10 01 05FF 000000]
20:21:19,546    DEBUG [ReceiverListener:hidraw1] logitech_receiver.base: (15) => r[10 01 05FF 000000]
20:21:19,570    DEBUG [ReceiverListener:hidraw1] logitech_receiver.base: (15) => r[10 01 05FF 000000]
20:21:19,594    DEBUG [ReceiverListener:hidraw1] logitech_receiver.base: (15) => r[10 01 05FF 000000]
20:21:21,842    DEBUG [ReceiverListener:hidraw1] logitech_receiver.base: (15) => r[10 01 0501 000000]
20:21:21,858    DEBUG [ReceiverListener:hidraw1] logitech_receiver.base: (15) => r[10 01 0501 000000]
20:21:21,874    DEBUG [ReceiverListener:hidraw1] logitech_receiver.base: (15) => r[10 01 0501 000000]
20:21:21,898    DEBUG [ReceiverListener:hidraw1] logitech_receiver.base: (15) => r[10 01 0501 000000]
20:21:22,074    DEBUG [ReceiverListener:hidraw1] logitech_receiver.base: (15) => r[10 01 0502 000000]
20:21:22,098    DEBUG [ReceiverListener:hidraw1] logitech_receiver.base: (15) => r[10 01 0501 000000]
20:21:22,114    DEBUG [ReceiverListener:hidraw1] logitech_receiver.base: (15) => r[10 01 0501 000000]
20:21:22,290    DEBUG [ReceiverListener:hidraw1] logitech_receiver.base: (15) => r[10 01 0502 000000]
20:21:22,322    DEBUG [ReceiverListener:hidraw1] logitech_receiver.base: (15) => r[10 01 0501 000000]
20:21:22,498    DEBUG [ReceiverListener:hidraw1] logitech_receiver.base: (15) => r[10 01 0501 000000]
20:21:22,514    DEBUG [ReceiverListener:hidraw1] logitech_receiver.base: (15) => r[10 01 0501 000000]


```

</details>

<p>
</details>

<p>
</details>

<details>
<summary>solaar probe</summary>

```
EX100 Receiver 27 Mhz
  Device path  : /dev/hidraw3
  USB id       : 046d:c517
  Serial       : None
  Has 2 paired device(s) out of a maximum of 4.
  Notifications: wireless (0x000100)
  Register Dump
    Notification Register 0x00: 0x000100
    Connection State      0x02: 0x000200
    Device Activity       0xb3: None
    Pairing Register 0xb5 0x00: None
    Pairing Register 0xb5 0x10: None
    Pairing Register 0xb5 0x20: None
    Pairing Register 0xb5 0x30: None
    Pairing Name     0xb5 0x40: None
    Pairing Register 0xb5 0x01: None
    Pairing Register 0xb5 0x11: None
    Pairing Register 0xb5 0x21: None
    Pairing Register 0xb5 0x31: None
    Pairing Name     0xb5 0x41: None
    Pairing Register 0xb5 0x02: None
    Pairing Register 0xb5 0x12: None
    Pairing Register 0xb5 0x22: None
    Pairing Register 0xb5 0x32: None
    Pairing Name     0xb5 0x42: None
    Pairing Register 0xb5 0x03: None
    Pairing Register 0xb5 0x13: None
    Pairing Register 0xb5 0x23: None
    Pairing Register 0xb5 0x33: None
    Pairing Name     0xb5 0x43: None
    Pairing Register 0xb5 0x04: None
    Pairing Register 0xb5 0x14: None
    Pairing Register 0xb5 0x24: None
    Pairing Register 0xb5 0x34: None
    Pairing Name     0xb5 0x44: None
    Pairing Register 0xb5 0x05: None
    Pairing Register 0xb5 0x15: None
    Pairing Register 0xb5 0x25: None
    Pairing Register 0xb5 0x35: None
    Pairing Name     0xb5 0x45: None
    Firmware         0xf1 0x00: None
    Firmware         0xf1 0x01: None
    Firmware         0xf1 0x02: None
    Firmware         0xf1 0x03: None
    Firmware         0xf1 0x04: None

```

</details>

![Slika zaslona 2020-08-28 20-24-53](https://user-images.githubusercontent.com/1978777/91603036-92fcc400-e96c-11ea-8b0b-ee524d5a15c4.png)

--- a/lib/logitech_receiver/device.py
+++ b/lib/logitech_receiver/device.py
@@ -78,17 +78,8 @@ class Device(object):
             if link_notification is not None:
                 self.online = not bool(ord(link_notification.data[0:1]) & 0x40)
                 self.wpid = _strhex(link_notification.data[2:3] + link_notification.data[1:2])
-                # assert link_notification.address == (0x04
-                # if unifying else 0x03)
+                # assert link_notification.address == (0x04 if unifying else 0x03)
                 kind = ord(link_notification.data[0:1]) & 0x0F
-                # fix EX100 wpid
-                if receiver.ex100_wpid_fix:  # EX100 receiver
-                    self.wpid = _strhex(link_notification.data[2:3]) + '00'
-                    # workaround for EX100 switched kind
-                    if self.wpid == '3F00':
-                        kind = 2
-                    if self.wpid == '6500':
-                        kind = 1
                 self._kind = _hidpp10.DEVICE_KIND[kind]
             else:
                 # force a reading of the wpid

--- a/lib/logitech_receiver/notifications.py
+++ b/lib/logitech_receiver/notifications.py
@@ -202,13 +202,9 @@ def _process_hidpp10_notification(device, status, n):
             0x0A else 'Lightspeed 1' if n.address == 0x0C else 'Lightspeed 1_1' if n.address == 0x0D else None
         )
         if protocol_name:
-            if _log.isEnabledFor(_DEBUG):
-                wpid = _strhex(n.data[2:3] + n.data[1:2])
-                # workaround for short EX100 wpids
-                if protocol_name == '27 MHz':
-                    wpid = _strhex(n.data[2:3]) + '00'
-                assert wpid == device.wpid, '%s wpid mismatch, got %s' % (device, wpid)
-
+            wpid = _strhex(n.data[2:3] + n.data[1:2])
+            if wpid != device.wpid:
+                _log.warn('%s wpid mismatch, got %s', device, wpid)
             flags = ord(n.data[:1]) & 0xF0
             link_encrypted = bool(flags & 0x20)
             link_established = not (flags & 0x40)

